Cargo.toml

```diff
@@ -10,7 +10,8 @@ exclude = ["packages/rust-lexicase"]
 version = "0.1.0"
 authors = ["Nic McPhee", "Many cool people on Twitch"]
```

Comment:
I feel like you ought to be listed as an author on the Push package. You've contributed a _lot_ to that, and a lot of that work is public, and I'd like to see you included there.

I'm also not entirely sure how one can/should best credit Twitch folks who have contributed, but just via the stream. Any thoughts you have on that would be very much appreciated.